src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -71,13 +72,20 @@ private static IEndpointConventionBuilder MapActorDeactivationEndpoint(this IEnd
         private static IEndpointConventionBuilder MapActorMethodEndpoint(this IEndpointRouteBuilder endpoints)
         {
             var runtime = endpoints.ServiceProvider.GetRequiredService<ActorRuntime>();
+            var logger = endpoints.ServiceProvider.GetRequiredService<ILoggerFactory>().CreateLogger<ILogger>();
```

Comment:
Is this used for anything?

Reply:
Nope, I had debugging code and removed it but missed the logger.